app/assets/javascripts/discourse-common/addon/helpers/i18n.js

```diff
@@ -1,6 +1,10 @@
 import I18n from "I18n";
 import { registerUnbound } from "discourse-common/lib/helpers";
 
+export default function i18n(key, params) {
+  return I18n.t(key, params);
+}
+
```

Comment:
@davidtaylorhq  Added this so I could use `{{i18n "summary.in_progress"}}` in the `.gjs` template.. Is this fine? Should I also add an export for `i18nYesNo` or can that wait till it needs to be used?

Reply:
Sounds good to me :+1:

`i18n-yes-no` doesn't seem commonly used, so let's ignore that for now. Looks like it's only used for the admin users section, so it should probably eventually end up as local function in the relevant file.